esphome/components/nextion/nextion.cpp

```diff
@@ -128,6 +131,7 @@ void Nextion::dump_config() {
   ESP_LOGCONFIG(TAG, "  Serial Number:    %s", this->serial_number_.c_str());
   ESP_LOGCONFIG(TAG, "  Flash Size:       %s", this->flash_size_.c_str());
   ESP_LOGCONFIG(TAG, "  Wake On Touch:    %s", this->auto_wake_on_touch_ ? "True" : "False");
+  ESP_LOGCONFIG(TAG, "  Exit reparse:     %s", this->exit_reparse_on_start_ ? "True" : "False");
```

Comment:
We have a macro for these:
```suggestion
  ESP_LOGCONFIG(TAG, "  Wake On Touch:    %s", YESNO(this->auto_wake_on_touch_));
  ESP_LOGCONFIG(TAG, "  Exit reparse:     %s", YESNO(this->exit_reparse_on_start_));
```